requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible-lint==3.5.1
+ansible-lint>=4.0.1
```

Comment:
Is there a specific reason we are setting the mimimum from `3.5.1` to `4.0.1`, rather than say
`ansible-lint>=3.5.1`

Reply:
Probably because we don't want linter to have too outdated behavior...